base/doc/ltnews42.tex

```diff
@@ -155,6 +155,23 @@ \subsection{Normalizing key names for block environments}
 
 \section{New or improved commands}
 
+\subsection{Support separate font families for script fonts}
+
+In \TeX's math processing separate fonts can be selected for text, script and
+scriptscript sizes. \LaTeX's NFSS traditionally uses this to load three sizes
+of the same font, handling adjustment needed for making fonts appear better in
+a script location through the use of optical sizes. This works great for traditional
+\TeX{} fonts, but for OpenType fonts this leads to issues. OpenType MATH assumes
+the font in a script location has separate features set and therefore received specific
+adjustments.
+
+To support this without relying on heuristics based on the font size, a new command
+\verb+\DeclareMathScriptfontMapping+ has been added. It can be called with three
+encodings and faily names to indicate that whenever the first family is selected as a
+math font, the second and thirs should be used as a script respectively scriptscript font.
```

Comment:
It takes 3 pairs of encoding/family arguments to indicate that for the first pair when used as the math main font the second and the third should be used as the script and scriptscript font, respectively.